features/reward_features.py

```diff
@@ -118,9 +122,7 @@ def _start_reward(self):
         self.reportstats['Reward #'] += 1
         
         if self.reportstats['Reward #'] % self.trials_per_reward == 0:
-            for _ in range(self.pellets_per_reward): # call trigger num of pellets_per_reward time
-                self.reward.trigger(self.ip_address)
-                time.sleep(0.5) # wait for 0.5 seconds
+            self.reward.trigger(self.ip_address, self.pellet_dispense_time, self.pellets_per_reward) # triggers as many times as pellets_per_reward
```

Comment:
to fix the manual reward button, you could include the ip_address as an input to the RemoteReward() init function. then you don't need to specify when you call self.reward.trigger()